Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -65,6 +65,32 @@ Additional Requirements for Native-routed Datapath Modes
   firewall rules allowing pods in different clusters to reach each other on all
   ports.
 
+Cluster Identity Requirements
```

Comment:
I'd personally suggest to shift the focus (both section title and first paragraph) from identities to the number of clusters that can be meshed together. That's what IMHO users are interested in reading, while the cluster identity mode/requirements look more an implementation detail. For instance, I'd swap the first and second sentence of the paragraph, and reword a bit the first one to just point out that the `maxConnectedClusters` value needs to be set to the same value across the whole clustermesh (mentioning the cluster identity mode seems superfluous to me).